governance/votes/Executive vote - November 26, 2021.md

```diff
@@ -0,0 +1,150 @@
+* [Maximum Auction Duration (`tail`)](https://community-development.makerdao.com/en/learn/governance/param-max-auction-duration): 90 minutes
+* [Breaker Price Tolerance (`tolerance`)](https://community-development.makerdao.com/en/learn/governance/param-breaker-price-tolerance): 0.5
+
+**Liquidation Incentives**
+
+* [Proportional Kick Incentive (`chip`)](https://community-development.makerdao.com/en/learn/governance/param-proportional-kick-incentive): 0.1%
+* [Flat Kick Incentive (`tip`)](https://community-development.makerdao.com/en/learn/governance/param-flat-kick-incentive): 300 DAI
+
+Please review the following forum threads containing information about WBTC-C to inform your position before voting.
+* [Proposal Thread](https://forum.makerdao.com/t/signal-request-new-vault-type-for-wbtc-with-a-higher-lr/11579)
+* [Risk Core Unit Evaluation](https://forum.makerdao.com/t/proposed-risk-parameters-for-wbtc-c-vault-type/11707)
+
+
+### GUSD Peg Stability Module (PSM) Onboarding
+
+As per the successful outcome of this [governance poll](https://vote.makerdao.com/polling/QmayeEjz?network=mainnet), PSM-GUSD-A will be onboarded with the parameters listed below if this executive proposal passes.
+
+**Proposed GUSD PSM Parameters**
+
+* [Fee In](https://makerdao.world/en/learn/governance/module-psm) (`tin`): 0%
+* [Fee Out](https://makerdao.world/en/learn/governance/module-psm) (`tout`): 0%
+* [Maximum Debt Ceiling](https://makerdao.world/en/learn/governance/module-dciam) (`line`): 10 million DAI\*
+* [Target Available Debt](https://makerdao.world/en/learn/governance/module-dciam) (`gap`): 10 million DAI
+* [Ceiling Increase Cooldown](https://makerdao.world/en/learn/governance/module-dciam) (`ttl`): 24 hours
+
+Please review the following threads to help inform your position before voting:
+* [Proposal](https://forum.makerdao.com/t/proposal-gusd-psm/11148)
+* [Risk Evaluation](https://forum.makerdao.com/t/psm-gusd-a-collateral-onboarding-risk-evaluation/11559)
+* [Technical Evaluation](https://forum.makerdao.com/t/gusd-erc20-token-smart-contract-technical-assessment/4603)
+* [Oracle Evaluation](https://forum.makerdao.com/t/mip10c3-sp13-proposal-gusd-oracle-collateral-onboarding-oracle-assessment/4777)
+
+\* Note that the initial Maximum Debt Ceiling is being set to 10 million DAI to limit risk. A proposal to increase the Maximum Debt Ceiling to 100 million will be included in a future executive vote.
+
+### Surplus Buffer Linear Increase
+
+As per the successful outcome of this [governance poll](https://vote.makerdao.com/polling/QmUqfZRv?network=mainnet#poll-detail), the Surplus Buffer will be increased $fill_in
+
+### Increase Aave D3M (DAI Direct Deposit Module) Debt Ceiling 
+
+As per the successful outcome of this [governance poll](https://vote.makerdao.com/polling/QmZhvNu5?network=mainnet), the Aave D3M Debt Ceiling will be increased from 50 million DAI to 100 million DAI if this executive proposal passes.
```

Comment:
Shouldn't it be "Maximum Debt Ceiling" as we're using an IAM?

Reply:
plus link to debt ceiling docs?

Reply:
Yes, already fixed this one.